extensionsdk/Microsoft.Windows.DevHome.SDK/Microsoft.Windows.DevHome.SDK.idl

```diff
@@ -144,7 +144,10 @@ namespace Microsoft.Windows.DevHome.SDK
     [contract(Microsoft.Windows.DevHome.SDK.DevHomeContract, 1)]
     enum AuthenticationState {
         LoggedIn = 0,
-        LoggedOut = 1
+        LoggedOut = 1,
+
+        [contract(Microsoft.Windows.DevHome.SDK.DevHomeContract, 2)] 
+        FixAccount = 2
```

Comment:
Reconsider the name. Maybe "NeedsRepair" or something similar. "FixAccount" doesn't sound like a state. :)